uax/uax42/Readme.md

```diff
@@ -0,0 +1,24 @@
+# Generating TR42
+
+## Step 1 - Generate property value fragments
+
+- Run org.unicode.xml.GeneratePropertyValues to populate the UNICODETOOLS_REPO_DIR/uax/uax42/fragments/ folder.
+
+## Step 2 - Generate TR42 index.html and index.rnc 
+
+- In UNICODETOOLS_REPO_DIR/uax/uax42/ run `mvn xml:transform`
+
+  index.html and index.rnc will be generated in UNICODETOOLS_REPO_DIR/uax/uax42/output/
```

Comment:
The output should go into a folder under UNICODETOOLS_GEN_DIR as usual, such as UNICODETOOLS_GEN_DIR/ucdxml/17.0.0/

Reply:
I think I have what you want, but...